cirq/ops/three_qubit_gates.py

```diff
@@ -145,6 +149,132 @@ def __str__(self) -> str:
         return 'CCZ**{}'.format(self._exponent)
 
 
+@value.value_equality()
+class ThreeQubitDiagonalGate(gate_features.ThreeQubitGate):
+    """A gate given by a diagonal 8x8 matrix.
+
+    The list of angles passed to the constructor should be in radians.
+    """
+
+    def __init__(self,
+                 diag_angles_radians: List[Union[float, sympy.Basic]]) -> None:
+        self._diag_angles_radians = diag_angles_radians  # type: List[float]
```

Comment:
This type annotation is incorrect. The actual type is List[Union[float, sympy.Basis]].

Reply:
Fixed.